src/cpr_sdk/yql_builder.py

```diff
@@ -155,6 +155,28 @@ def build_year_end_filter(self) -> Optional[str]:
                 return f"(family_publication_year <= {end})"
         return None
 
+    def build_concept_count_filter(self) -> Optional[str]:
+        """Create the part of the query that filters on concept counts"""
+        concept_count_filters_subqueries = []
+        if self.params.concept_count_filters:
+            for concept_count_filter in self.params.concept_count_filters:
+                concept_count_filters_subqueries.append(
+                    f"""
+                    (
+                        concept_counts contains sameElement(
+                            {(
+                                f'key contains "{concept_count_filter.concept_id}", '
+                                if concept_count_filter.concept_id is not None else ""
+                            )}
+                            value {concept_count_filter.operand.value} {concept_count_filter.count}
+                        )
+                    )
+                    """
+                )
+
+            return f"({' and '.join(concept_count_filters_subqueries)})"
```

Comment:
Not to spark the and or debate but which do we want here or do we want to expose the operand to the user (e.g. frontend dev, data scientist etc...)

Reply:
We may not know enough now, and hopefully would discover through people using it